src/product.py

```diff
@@ -7,12 +7,30 @@ class Product:
     # Свойства класса.
     name: str
     description: str
-    price: float
+    __price: float
     quantity: int
 
     def __init__(self, name, description, price, quantity):
         # Инициализация класса, передача значений атрибутов класса.
         self.name = name
         self.description = description
-        self.price = price
+        self.__price = price
```

Comment:
Иначе при инициализации объекта, сеттер не отработает и можно будет создать продукт с отрицательной ценой, что не верно `Product(..., price=-10)`. Нужно поправить  + тест
```suggestion
        self.price = price
```

Reply:
Добавил обработчик ошибки: если цена меньше или равна 0:
self.__price = price if price > 0 else ValueError("Цена не должна быть нулевая или отрицательная")